src/js/ttsHookup.js

```diff
@@ -50,10 +50,19 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                         speakPanelMessage: {
                             funcName: "gpii.firstDiscovery.tts.fdHookup.speakPanelMessage",
                             args: ["{firstDiscoveryEditor}", "{that}.msgLookup.panelMsg", "{that}.queueSpeech"]
+                        },
+                        speakPanelInstructions: {
+                            funcName: "gpii.firstDiscovery.tts.fdHookup.speakPanelInstructions",
+                            args: ["{firstDiscoveryEditor}", "{that}.queueSpeech"]
                         }
                     },
                     listeners: {
-                        "onCreate.readPanel": "{that}.speakPanelMessage"
+                        "onCreate.readPanel": "{that}.speakPanelMessage",
+                        "onCreate.bindKeypress": {
+                            listener: "gpii.firstDiscovery.tts.fdHookup.bindKeypress",
```

Comment:
`fluid.activatable` sets uses "space" and "enter" to activate. We can add additional bindings, but we wouldn't want "space" and "enter" to trigger reading.

https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/jquery.keyboard-a11y.js#L600-L608